apps/web-giddh/src/app/shared/create-address/create-address.component.html

```diff
@@ -53,25 +53,33 @@ <h3 *ngIf="addressConfiguration.type === 'editWarehouse'">{{ localeData?.edit_wa
                                         </div>
 
                                         <div class="row">
-                                            <div *ngIf="addressConfiguration?.tax?.name" class="col-sm-4 form-group">
+                                            <div *ngIf="isBusinessTypeRegistered" class="col-sm-4 form-group">
```

Comment:
confirm with @ravindersinghdang sir

Reply:
@dvCodeWorld this won't work if the index of tax number is 0.